### DISCUSSION TITLE
What do you think of the modified data?

### DISCUSSION THREAD

**Discussion by King-Darius** [King-Darius/XLM_Rongorongo#4] [Q&A] [answered]
[Tell me what you think of this before I add it as an alternative option.](https://github.com/King-Darius/XLM_Rongorongo/tree/Unique_Identifiers/data_preparation/encoding) @jgregoriods I am a bit new to this.

**Comment by King-Darius** [comment]
I think that the it cycled back through the list of characters, so the code needs updated and the output too.

**Comment by King-Darius** [comment]
The code should be changed to have the following: 
'α', 'β', 'γ', 'δ', 'ε', 'ζ', 'η', 'θ', 'ι', 'κ', 'λ', 'μ', 'ν', 'ξ', 'ο', 'π', 'ρ', 'σ', 'τ', 'υ', 'φ', 'χ', 'ψ', 'ω', 'А', 'Б', 'В', 'Г', 'Д', 'Е', 'Ж', 'З', 'И', 'Й', 'К', 'Л', 'М', 'Н', 'О', 'П', 'Р', 'С', 'Т', 'У', 'Ф', 'Х', 'Ц', 'Ч', 'Ш', 'Щ', 'Ъ', 'Ы', 'Ь', 'Э', 'Ю', 'Я','τ', 'в', 'С', 'Н', 'Γ', 'Π', 'Ц', 'р', 'ν', 'У', 'ь', 'З', 'Ξ', 'т', 'Χ', 'э', 'Φ', 'Κ', 'п', 'Μ', 'Ρ', 'Т', 'о', 'ϒ', 'Η', 'γ', 'и', 'Β', 'с', 'α', 'ъ', 'Й', 'Ι', 'л', 'д', 'г', 'ψ', 'Ъ', 'Δ', 'а', 'Υ', 'Д', 'Ν', 'ю', 'ж', 'π', 'Л', 'у', 'я', 'Ζ', 'Щ', 'ф', 'ς', 'ξ', 'ο', 'Θ', 'е', 'μ', 'П', 'υ', 'м', 'Ε', 'χ', 'Ы', 'λ', 'Ч', 'й', 'ω', 'ι', 'В', 'ц', 'κ', 'Я', 'б', 'О', 'М', 'Λ', 'Ж', 'Ю', 'Э', 'з', 'х', 'Е', 'Р', 'н', 'ы', 'Ь', 'β', 'И', 'θ', 'Х', 'Σ', 'Ψ', 'ч', 'Ш', 'щ', 'φ', 'к', 'ε', 'Ф', 'η', 'Г', 'ζ', 'К', 'ш', 'Ο'

**Comment by King-Darius** [comment]
Also found this on HuggingFace [XLM-R](https://huggingface.co/FacebookAI/xlm-roberta-base).

**Comment by jgregoriods** [comment, marked as answer]
Sorry, I haven't had much time to work on this. I think we can experiment with different solutions, including yours. Another possibility is to just keep the original code when the reading is uncertain.

I assume we also need a Rapanui language sample, correct?

**Reply by King-Darius** [reply]
No worries!  Yes experimenting with different approaches seems best.  We need Rapa Nui samples and I imagine other polynesian languages.  

I also don't know yet how we ge from Unsupervised Cross-lingual Representation to Unsupervised machine translation using XLM-R.  Any ideas?
